src/test/store/profile-view.test.js

```diff
@@ -609,6 +609,17 @@ describe('actions/ProfileView', function() {
     });
   });
 
+  describe('changeNetworkSearchString', function() {
+    it('changes the search string', function() {
+      const profile = getProfileWithMarkers([['a', 0, null], ['b', 1, null]]);
```

Comment:
So this doesn't actually create a profile with network markers. If you search around there are some functions that will do this in the tests. Something like `getProfileWithNetworkMarkers`. Although here, it would probably be fine just to have an empty profile, as we aren't checking that the markers are filtered correctly.